substrate/frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -189,6 +189,13 @@
 //! Note that there could be an overlap between these sub-errors. For example, A
 //! `SnapshotUnavailable` can happen in both miner and feasibility check phase.
 //!
+//!	## Multi-page election support
+//!
+//!	Even though the [`frame_election_provider_support::ElectionDataProvider`] and
+//!	[`frame_election_provider_support::ElectionProvider`] traits support multi-paged election, this
+//!	pallet only supports a single page election flow. Thus, [`Config::Pages`] must be always set to
+//!	one, which is asserted by the [`frame_support::traits::Hooks::integrity_test`].
```

Comment:
So in short this pallet does not support multi-page election and if one wants to the multi-page election on their runtime, they should use the multi-block pallet, correct?
If it's so it'd be good to add such a mention to the doc.